packages/ai/package.json

```diff
@@ -24,20 +21,12 @@
   },
   "exports": {
     ".": {
-      "webstudio": "./src/index.ts",
-      "types": "./lib/types/index.d.ts",
-      "import": "./lib/index.js"
+      "webstudio": "./src/index.ts"
```

Comment:
that looks a little weird, isn't this an entry point name? why is the name webstudio? is it like "main"?

Reply:
it is our custom condition, we use it to reference sources within monorepo

Reply:
in that case maybe there is a convetion like with "@" or just call it "monorepo"?

Reply:
convention is within project to not conflict with other projects